sources/packages/api/src-sql/Restrictions/Rollback-truncate-restriction.sql

```diff
@@ -0,0 +1,255 @@
+-- Insert old restrictions data to the sims.restrictions table.
+INSERT INTO
+  sims.restrictions(
+    restriction_type,
+    restriction_code,
+    description,
+    restriction_category
+  )
+VALUES
+  (
+    'Federal',
+    '4',
+    '# of weeks >= 468, input by CSLP',
+    'Federal'
+  );
+
+INSERT INTO
+  sims.restrictions(
+    restriction_type,
+    restriction_code,
+    description,
+    restriction_category
+  )
+VALUES
+  (
+    'Federal',
+    '5',
+    '# of weeks >= 520, input by CSLP',
+    'Federal'
+  );
+
+INSERT INTO
+  sims.restrictions(
+    restriction_type,
+    restriction_code,
+    description,
+    restriction_category
+  )
+VALUES
+  (
+    'Federal',
+    '6',
+    '# of weeks >= 288, input by CSLP',
+    'Federal'
+  );
+
+INSERT INTO
+  sims.restrictions(
+    restriction_type,
+    restriction_code,
+    description,
+    restriction_category
+  )
+VALUES
+  (
+    'Federal',
+    '7',
+    '# of weeks >= 340, input by CSLP',
+    'Federal'
+  );
+
+INSERT INTO
+  sims.restrictions(
+    restriction_type,
+    restriction_code,
+    description,
+    restriction_category
+  )
+VALUES
+  (
+    'Federal',
+    'A',
+    'Fed, near maximum, Type A',
+    'Federal'
+  );
+
+INSERT INTO
+  sims.restrictions(
+    restriction_type,
+    restriction_code,
+    description,
+    restriction_category
+  )
+VALUES
+  (
+    'Federal',
+    'BO',
+    'Federal, near maximum, Type B',
+    'Federal'
+  );
+
+INSERT INTO
+  sims.restrictions(
+    restriction_type,
+    restriction_code,
+    description,
+    restriction_category
+  )
+VALUES
+  (
+    'Federal',
+    'C',
+    'Standard claim, input by CSLP',
+    'Federal'
+  );
+
+INSERT INTO
+  sims.restrictions(
+    restriction_type,
+    restriction_code,
+    description,
+    restriction_category
+  )
+VALUES
+  (
+    'Federal',
+    'D',
+    'Death (History Code)',
+    'Federal'
+  );
+
+INSERT INTO
+  sims.restrictions(
+    restriction_type,
+    restriction_code,
+    description,
+    restriction_category
+  )
+VALUES
+  (
+    'Federal',
+    'I',
+    'Permanent disability, input by CSLP',
+    'Federal'
+  );
+
+INSERT INTO
+  sims.restrictions(
+    restriction_type,
+    restriction_code,
+    description,
+    restriction_category
+  )
+VALUES
+  (
+    'Federal',
+    'O',
+    'Students has rec''d O/A-input by Province',
```

Comment:
Typo here. Please remove the second s from Students. I think it should be "Student has rec''d O/A-input by Province".

Reply:
think is actually rollback data (old data), which is the same as the initial one, 
![image](https://user-images.githubusercontent.com/77353155/168696142-46cab609-d93d-4903-ac2f-d8176b14bc3e.png)

Reply:
The intention is to insert the string value which itself has a  '. In this case it is rec'd.